scripts/process_jartic_parallel.py

```
@@ -105,8 +105,12 @@ def process_prefecture_data(pref_data_tuple):
         return pref_zip_name, None, 0
 
 
-def process_archive_parallel(archive_path: Path, num_workers: int = None):
-    manager = ExternalDataManager()
+def process_archive_parallel(archive_path: Path, num_workers: int = None, custom_data_dir: str = None):
```

Ruff (0.12.2)

`process_archive_parallel` is too complex (17 > 10)

(C901)